README.md

```diff
@@ -1,59 +1,33 @@
-# Balena Weather
+# Balena Weather Station
 
-A Raspberry Pi weather station, implemented as [Balena](https://www.balena.io/) multi-container application.
+A Raspberry Pi weather station, running on a [Balena](https://www.balena.io/) multi-container application where you can add more services. This project is inspired by the [Raspberry Pi weather station](https://projects.raspberrypi.org/en/projects/build-your-own-weather-station/0).
 
-[![balena deploy button](https://www.balena.io/deploy.svg)](https://dashboard.balena-cloud.com/deploy?repoUrl=https://github.com/hferentschik/balena-weather)
-
-![Grafana Dash](./images/dash.png)
-
-Raspberry Pi 3                                | Weather station
```

Comment:
A good one to break up this table. It was an attempt to get the two pictures next to each other, but depending on your device it does not work as expected.